models/codecs.py

```diff
@@ -20,11 +20,17 @@
 
         "businessCustomer": lambda _id, _type, attributes, relationships:
         BusinessCustomerDTO.from_json_api(_id, _type, attributes, relationships),
+
+        "depositAccount": lambda _id, _type, attributes, relationships:
+        DepositAccountDTO.from_json_api(_id, _type, attributes, relationships),
+
+        "limits": lambda _id, _type, attributes, relationships:
+        AccountLimitsDTO.from_json_api(_type, attributes),
     }
 
 
 def split_json_api_single_response(payload: dict):
-    _id, _type, attributes = payload["id"], payload["type"], payload["attributes"]
+    _id, _type, attributes = payload.get("id"), payload["type"], payload["attributes"]
```

Comment:
there is a situation where id can be None?

Reply:
https://developers.unit.co/#account-limits doesn't have an id